node/cli/src/command.rs

```diff
@@ -386,69 +404,24 @@ pub fn run() -> Result<()> {
 				Ok((cmd.run(version), task_manager))
 			})
 		},
-		None => {
-			let runner = cli.create_runner(&cli.run.normalize())?;
-
-			runner.run_node_until_exit(|config| async move {
-				let para_id = chain_spec::Extensions::try_get(&*config.chain_spec)
-					.map(|e| e.para_id)
-					.ok_or("Could not find parachain ID in chain-spec.")?;
-
-				#[cfg(feature = "frequency-rococo-local")]
-				if cli.instant_sealing {
-					return frequency_service::service::frequency_dev_instant_sealing(config, true)
-						.map_err(Into::into)
-				} else if cli.manual_sealing {
```

Comment:
This will be addressed in the next issue I am working on to add timed sealing.  I intend to replace these booleans with a single enum for sealing mode.